scripts/security-updates/automatic-update

```diff
@@ -230,7 +229,84 @@ if [[ $(security_updates_available) == "true" ]]; then
         echo "Zendesk Ticket successfully created."
         echo "-------------------------------------------"
     fi
+
+    # Create a ticket for ClickUp.
+    if [ -n "${CU_REQUESTER_ID}" -a -n "${CU_TOKEN}" -a -n "${CU_LEAD_EMAIL}" ]; then
+
+      # Creating ClickUp ticket.
+      echo "Creating security updates ClickUp Ticket."
+      echo "-------------------------------------------"
+      CU_API=https://api.clickup.com/api/v2
+      # Environment variables; should be defined as CircleCI variables.
+      # CU_CLIENT_ID: Look in ClickUp for id. e.g. 16
+      # CU_REQUESTER_ID: Look in ClickUp for id. e.g. 54648604
+      # CU_LIST_ID: Look in ClickUp for id. e.g. 901300865356
+      # CU_TOKEN: Located in 1Pass.
+      # CU_LEAD_EMAIL: Find lead in engagements database.
+
+      CU_BODY_MESSAGE=`cat ./vendor/fourkitchens/pots/scripts/security-updates/TICKET_TEMPLATE.md`
+      # Replacing placeholders in the template.
+      CU_BODY_MESSAGE=${CU_BODY_MESSAGE//\[SITE_NAME\]/$SITE_NAME}
+      CU_BODY_MESSAGE=${CU_BODY_MESSAGE//\[PROJECTS\]/$PROJECTS_UPDATED}
+      # Scraping new line characters.
+      CU_BODY_MESSAGE=${CU_BODY_MESSAGE//$'\n'/\\n}
+      CU_TICKET_TITLE="Security Updates Required For Your Website - ${SITE_NAME}"
+      CU_JSON_DATA=$(printf '{
+                               "name": "%s",
+                               "assignees": [
+                                 82036728
+                               ],
+                               "tags": [
+                                 "autosec update"
+                               ],
+                               "status": "in progress",
+                               "priority": 2,
+                               "start_date": "%s",
+                               "start_date_time": false,
+                               "notify_all": false,
+                               "parent": null,
+                               "links_to": null,
+                               "check_required_custom_fields": true,
+                               "custom_fields": [
+                                 {
+                                   "id": "b8bbcb75-a7c8-4eb7-9202-e56d227eefbf",
+                                   "value": "%s"
+                                 }
+                               ],
+                               "description": "%s"
+                             }' "$CU_TICKET_TITLE" "$UNIX_TIMESTAMP_MS" "$CU_CLIENT_ID" "$CU_BODY_MESSAGE")
+
+        # Create the ticket and capture the response.
+        RESPONSE=$(
+            curl -s ${CU_API}/list/901300865356/task \
+              -i -X POST \
+              -H "Authorization: ${CU_TOKEN}" \
+              -H "Content-Type: application/json" \
+              -d "$CU_JSON_DATA"
+          )
+
+        # Extract the task id from the response; I could not get jq to work.
```

Comment:
Likely would need to be installed. It may exist on the Pantheon docker image and not on the CircleCI base images.

Reply:
Also, consider using PHP here in the future. You could json_decode and grab the TASK_ID. You can see where I've done something similar here: https://github.com/fourkitchens/acquia-cloud-hooks/blob/main/helper/get-env-uuid.php

Reply:
This is creating tickets. You can see the one's it created on the "In Progress" section for our test: https://app.clickup.com/36718269/v/b/6-901300865356-2

I think Rodrigo was trying to get something else done before merging this in.